php/class-wp-cli.php

```diff
@@ -769,6 +769,25 @@ public function create_guest_authors_from_wxr( $args, $assoc_args ) {
 		WP_CLI::line( 'All done!' );
 	}
 
+	/**
+	 * Create a single guest author.
+	 *
+	 * self::create_guest_author() wrapper.
+	 *
+	 * @subcommand create-single-author
```

Comment:
Can we do `create-author`?

Reply:
Thanks for reviewing @rebeccahum , for sure I just updated my commit here https://github.com/Automattic/Co-Authors-Plus/pull/880/commits/c9d26853a3f19363c90c324fe6210b9be6b5350a